chainer/variable.py

```diff
@@ -1320,6 +1324,10 @@ def initialize(self, shape):
         self.data = data
         self.grad = grad
 
+        # Convert the array for iDeep.
```

Comment:
https://github.com/chainer/chainer/pull/4457/files#diff-e131a8dfc2ea641e7e7c63597b9b0d8eR1316
Don't we have to check `_initial_backend` here?
I understand it works without checking, but `_initial_device` is now subsidiary concept of `_initlal_backend`, so I think whenever we use `_initial_device`, we should check `_initial_backend` first.

Reply:
Fixed to use `_initial_backend`.